src/tasks/daprdCommandTaskProvider.ts

```diff
@@ -59,7 +59,7 @@ export default class DaprdCommandTaskProvider extends CommandTaskProvider {
                                 .withNamedArg('--alsologtostderr', daprDefinition.alsoLogToStdErr, { assignValue: true })
                                 .withNamedArg('--app-id', daprDefinition.appId)
                                 .withNamedArg('--app-port', daprDefinition.appPort)
-                                .withNamedArg('--components-path', daprDefinition.componentsPath)
+                                .withNamedArg('--components-path', daprDefinition.componentsPath ?? './components')
```

Comment:
```suggestion
                                .withNamedArg('--components-path', daprDefinition.componentsPath || './components')
```
Unless `''` (empty string) is a valid value, I'd use `||` instead of `??`.

Reply:
Since we're basically acting as a pass-through, I'm inclined to not do too much interpretation of the user's intent and provide a default only when the user actually omits a value, allowing the underlying CLI to determine what `''` ultimately represents.